### ISSUE TITLE
Catmandu export support --id option to read identifiers from a file

### ISSUE THREAD

**Issue by phochste** [LibreCat/Catmandu#342]
```
 catmandu export MongoDB --bag pubs --id /tmp/ids.txt to YAML
```

**Comment by nics** [comment]
The id option already accepts an array of values, just make a space separated file and then call:

    catmandu export MongoDB --bag pubs --id `cat /tmp/ids.txt` to YAML

If you need to export a really huge number of ids, we can add a new --id-file option?

**Comment by nics** [comment]
implemented in 4317ad3323a66d0b906339fde4c69a92d2bbe140
